nighthawk/test/test_data/benchmark_http_client_test_envoy.yaml

```diff
@@ -0,0 +1,49 @@
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: {{ ip_loopback_address }}
+      port_value: 0
+static_resources:
+  clusters:
+    name: cluster_0
+    hosts:
+      socket_address:
+        address: {{ ip_loopback_address }}
+        port_value: 0
+  listeners:
+    - name: listener_0
+      address:
+        socket_address:
+          address: {{ ip_loopback_address }}
+          port_value: 0
+      filter_chains:
+      - filters:
+        - name: envoy.http_connection_manager
+          config: 
+            generate_request_id: false
+            codec_type: auto
+            stat_prefix: ingress_http
+            route_config:
+              name: local_route 
+              virtual_hosts:
+              - name: service
+                domains:
+                - "*"
+                routes:
+                - match:
+                    prefix: /lorem-ipsum-status-200
+                  direct_response:
+                    status: 200
+                    body:
+                      filename: nighthawk/test/test_data/lorem_ipsum.txt
```

Comment:
Interesting that this works. I think we probably should do a runfiles based substitution here though, since we will need this when we import to Google (the monorepo path will have some additional prefix that we need a hook to substitute).